clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -831,10 +823,9 @@ abstract class RetriableRequestState extends RequestState {
          * Complete the request future with a TimeoutException if the request timeout has been
          * reached, based on the provided current time.
```

Comment:
Let's update this to include the "if the request has been sent out at least once and...(timeout expired)"

Reply:
Updated. PTAL.